tests/test-tty

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
```

Comment:
Can you add some kind of docs to this test? It's not clear to me at all what is being tested here.